### PR TITLE
modify timestamps: ongoing. I'm still generating new datasets.

### PR DESCRIPTION
**Pull request by TTTTao725**

### CODE REVIEW

**Comment by TTTTao725** [comment]
All scripts are done, currently generating new datasets

**Generating:**
- [ ] dagw
- [ ] memo
- [ ] infomedia
- [ ] scrape_hovedstaden
- [ ] swedish_gigaword

**Testing using dataset validator:**
- [ ] dagw
- [ ] memo
- [ ] infomedia
- [ ] scrape_hovedstaden
- [ ] swedish_gigaword

File: data-processing/scripts/dataset_validator.py [hunk 1]
```diff
@@ -80,7 +79,8 @@ def check_timestamp_format(cls, v: str):  # noqa
         if not v:
             raise ValueError("Timestamp 'added' is required.")
         try:
-            datetime.strptime(v, "%Y-%m-%dT%H:%M:%S.%fZ")
+            # datetime.strptime(v, "%Y-%m-%dT%H:%M:%S.%fZ")
```
Comment (KennethEnevoldsen) [inline data-processing/scripts/dataset_validator.py:82]:
```suggestion
```

File: data-processing/scripts/dataset_validator.py [hunk 2]
```diff
@@ -97,11 +97,13 @@ def check_created_format(cls, v: str):
...
             if start_date > end_date:
                 raise ValueError(
-                    "Timestamp 'created' should be in the format 'YYYY-MM-DDTHH:MM:SS.TIMEZONE, YYYY-MM-DDTHH:MM:SS.TIMEZONE'.",
+                    #"Timestamp 'created' should be in the format 'YYYY-MM-DDTHH:MM:SS.TIMEZONE, YYYY-MM-DDTHH:MM:SS.TIMEZONE'.",
+                    "Timestamp 'created' should be in the format 'YYYY-MM-DD, YYYY-MM-DD'.",
                 )
         except ValueError as e:
             raise ValueError(
-                "Timestamp 'created' should be in the format 'YYYY-MM-DDTHH:MM:SS.TIMEZONE, YYYY-MM-DDTHH:MM:SS.TIMEZONE'. Got additional error:\n"
+                # "Timestamp 'created' should be in the format 'YYYY-MM-DDTHH:MM:SS.TIMEZONE, YYYY-MM-DDTHH:MM:SS.TIMEZONE'. Got additional error:\n"
```
Comment (KennethEnevoldsen) [inline data-processing/scripts/dataset_validator.py:105]:
```suggestion
```

File: data-processing/scripts/convert_dagw_to_jsonlgz.py [hunk 3]
```diff
@@ -59,31 +123,59 @@ def reformat_and_clean_dataset(ds: Dataset, num_proc: int) -> Dataset:
...
+    #     "opensub": "2000-01-01T00:00:00.000Z, 2022-01-01T00:00:00.000Z",
+    #     "relig": "NA",
+    #     "spont": "2019-01-01T00:00:00.000Z, 2020-01-01T00:00:00.000Z",
+    #     "synne": "2000-01-01T00:00:00.000Z, 2022-01-01T00:00:00.000Z",
+    #     "tv2r": "2015-01-01T00:00:00.000Z, 2020-01-01T00:00:00.000Z",
+    #     "wiki": "2019-01-01T00:00:00.000Z, 2021-01-01T00:00:00.000Z",
+    #     "wikibooks": "2019-01-01T00:00:00.000Z, 2021-01-01T00:00:00.000Z",
+    #     "wikisource": "1700-01-01T00:00:00.000Z, 2022-01-01T00:00:00.000Z",
+    #     "twfv19": "2000-01-01T00:00:00.000Z, 2022-01-01T00:00:00.000Z",  # not present in this version of the dataset
+    # }
```
Comment (KennethEnevoldsen) [inline data-processing/scripts/convert_dagw_to_jsonlgz.py:151]:
```suggestion
```

**Review by KennethEnevoldsen — APPROVED** [review]
A few places where you have code that is commented out. I would just delete it. Otherwise I believe this looks good!

**Comment by TTTTao725** [comment]
Yes, I was just a bit worried that we might need it somehow in the future, I'll clean it up after all the datasets are done (hopefully tomorrow)

**Pull request merged**